templates/dns.yml.erb

```diff
@@ -17,7 +16,10 @@
 <%if scope.lookupvar("foreman_proxy::dns_provider") == 'nsupdate_gss' -%>
 :dns_tsig_keytab: <%= scope.lookupvar("foreman_proxy::dns_tsig_keytab") %>
 :dns_tsig_principal: <%= scope.lookupvar("foreman_proxy::dns_tsig_principal") %>
+# dns_key must be disabled if nsupdate_gss is used
+#:dns_key: <%= scope.lookupvar("foreman_proxy::keyfile") %>
```

Comment:
According to the docs, ```:dns_key: false``` should be set. It makes probably no difference -proxy wise, but please change it to align it with the docs.